pkgs/by-name/po/portmaster/package.nix

```
@@ -0,0 +1,171 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  makeWrapper,
+  glib,
+  gtk3,
+  cairo,
+  pango,
+  gdk-pixbuf,
+  atk,
+  webkitgtk_4_1,
+  libsoup_3,
+  openssl,
+  curl,
+  systemd,
+  iptables,
+  iproute2,
+  xorg,
+  alsa-lib,
+  nss,
+  nspr,
+  at-spi2-atk,
+  cups,
+  dbus,
+  expat,
+  fontconfig,
+  freetype,
+  zlib,
+  unzip,
+  libayatana-appindicator,
+  coreutils,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
a bit more convenient and obvious for recursion (you can see, where attrs come from + you can reuse implicit attrs)